modules/xnft/src/impl_transactor.rs

```diff
@@ -0,0 +1,134 @@
+// This file is part of Acala.
+
+// Copyright (C) 2023 Unique Network.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use crate::*;
+
+const LOG_TARGET: &str = "xcm::module_xnft::transactor";
+
+impl<T: Config> TransactAsset for Pallet<T>
+where
+	TokenIdOf<T>: TryFrom<u128>,
+	ClassIdOf<T>: TryFrom<u128>,
+{
+	fn can_check_in(
+		_origin: &xcm::v3::MultiLocation,
+		_what: &MultiAsset,
+		_context: &xcm::v3::XcmContext,
+	) -> xcm::v3::Result {
+		Err(xcm::v3::Error::Unimplemented)
+	}
+
+	fn check_in(_origin: &xcm::v3::MultiLocation, _what: &MultiAsset, _context: &xcm::v3::XcmContext) {}
+
+	fn can_check_out(
+		_dest: &xcm::v3::MultiLocation,
+		_what: &MultiAsset,
+		_context: &xcm::v3::XcmContext,
+	) -> xcm::v3::Result {
+		Err(xcm::v3::Error::Unimplemented)
+	}
+
+	fn check_out(_dest: &xcm::v3::MultiLocation, _what: &MultiAsset, _context: &xcm::v3::XcmContext) {}
+
+	fn deposit_asset(what: &MultiAsset, who: &xcm::v3::MultiLocation, _context: &xcm::v3::XcmContext) -> XcmResult {
```

Comment:
context используется в логах

```suggestion
	fn deposit_asset(what: &MultiAsset, who: &xcm::v3::MultiLocation, context: &xcm::v3::XcmContext) -> XcmResult {
```